CHANGELOG.md

```diff
@@ -3,6 +3,16 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
```

Comment:
You don't need the added/removed sections unless there is something to actually go in them.

Reply:
okay. will remove then